packages/contracts/deploy-config/devnetL1-template.json

```diff
@@ -37,16 +37,15 @@
   "l2GenesisDeltaTimeOffset": "0x0",
   "l2GenesisEcotoneTimeOffset": "0x2",
   "systemConfigStartBlock": 0,
-
   "validatorRewardScalar": 5000,
   "validatorPoolTrustedValidator": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
   "validatorPoolRequiredBondAmount": "0x1",
   "validatorPoolMaxUnbond": 10,
   "validatorPoolRoundDuration": 4,
-  "validatorPoolTerminateOutputIndex": "0xf",
+  "validatorPoolTerminateOutputIndex": "0xffffffffffffffffffff",
   "validatorManagerTrustedValidator": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
-  "validatorManagerMinRegisterAmount": "0x1",
-  "validatorManagerMinActivateAmount": "0x2",
+  "validatorManagerMinRegisterAmount": "0x5",
+  "validatorManagerMinActivateAmount": "0xa",
```

Comment:
Since tax rate is 20% and slashing rate is 50%, if min activate amount is less than 10, tax is calculated as 0. So I increased it to test properly.